.github/workflows/patch-release.yml

```diff
@@ -0,0 +1,24 @@
+name: patch release
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 0 1 * *'
+env:
+  GO_VERSION: '1.22'
+jobs:
+  patch-release:
+    permissions:
+      contents: none
+    if: github.ref == 'refs/heads/main'
```

Comment:
This needs to be run on `release-*` branches rather than `main`.

Reply:
updated this change as requested